common/src/main/java/indexingTopology/common/data/DataSchema.java

```diff
@@ -222,4 +251,78 @@ public boolean equals(Object object) {
 
         return true;
     }
+
+
+    public DataTuple parseJsonTuple(String tuple, String split) {
+        String[] attributes = tuple.split(split);
+        DataTuple dataTuple = new DataTuple();
+        StringBuffer stringBuffer = new StringBuffer();
+        SimpleDateFormat jsonfmt = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+        Object attribute;
+        for (int i = 0; i < attributes.length; i++) {
+            Date date = null;
+            try {
+                date = jsonfmt.parse(attributes[i]);
```

Comment:
You don't need to parse Date, since Date is not supported by DataSchema. Date field should be defined as String by the user.